client/linuxabi/linux_abi.go

```diff
@@ -106,87 +112,185 @@ type SnpReportReq struct {
 	reserved [28]byte
 }
 
-// SnpReportResp is Linux's sev-guest ioctl abi for receiving a GET_REPORT response.
-type SnpReportResp struct {
+// SnpReportRespABI is Linux's sev-guest ioctl abi for receiving a GET_REPORT response.
+type SnpReportRespABI struct {
+	Status     uint32
+	ReportSize uint32
+	reserved   [0x20 - 8]byte
 	// Data is the response data, see SEV-SNP spec for the format
-	Data [abi.ReportSize]uint8
```

Comment:
Why did we have to change this?

Reply:
For the ABI to be allowed to use the full size that it expects at the boundary.